utils/constants.ts

```diff
@@ -7,6 +7,13 @@ export namespace ERC20TokenSymbol {
     export const MKR = "MKR";
 }
 
+/**
+ * An estimate of Ethereum blocktime: The typical number of sections between each block.
```

Comment:
I'm not sure I understand what this means; this is denominated in seconds, no?

Reply:
Yes it's in seconds. So, how long it takes for a block to be mined.

Reply:
More specifically, did you mean to say "seconds" instead of "sections"?